media_driver/linux/gen12/ddi/media_libva_caps_g12.cpp

```diff
@@ -1642,7 +1642,7 @@ VAStatus MediaLibvaCapsG12::CreateEncAttributes(
     if (entrypoint != VAEntrypointEncSliceLP ||
             (entrypoint == VAEntrypointEncSliceLP && MEDIA_IS_SKU(&(m_mediaCtx->SkuTable), FtrEnableMediaKernels)))
     {
-        attrib.value |= VA_RC_CBR | VA_RC_VBR | VA_RC_MB;
+        attrib.value |= VA_RC_CBR | VA_RC_VBR | VA_RC_MB | VA_RC_TCBRC;
```

Comment:
Is TCBRC really supported by both VDEnc and VME for all encoders?

Reply:
It is AVC/HEVC VDENC only

Reply:
Corresponding HEVC update here https://github.com/intel-innersource/drivers.gpu.unified/pull/879